src/utils/__tests__/tree-utils.spec.js

```diff
@@ -18,6 +18,16 @@ describe('tree-utils', () => {
       const node = createNodes(matrix, [], null, null, translator);
       expect(node.id).to.equal('007');
     });
+    it('should add a warning for max data', () => {
+      const matrix = [[{ qText: '007', qElemNumber: 0 }, { qText: '-' }]];
+      const node = createNodes(matrix, [], 'max-data-limit', null, translator);
+      expect(node.warn).to.eql(['Object.OrgChart.MaxData']);
```

Comment:
Does this function work?

Reply:
equal is equivalent to `===`, which would return false, since it is not the same instance of the array. So `eql` checks the contents instead, kind of... https://stackoverflow.com/questions/36798993/what-is-the-difference-between-equal-and-eql-in-chai-library

Reply:
Ah nice, I have been using deep equal I think :)

Reply:
It is very similar, didn't really understand the difference when I read about it